src/NServiceBus.AcceptanceTests/Core/TransportSeam/When_transport_is_started.cs

```diff
@@ -70,6 +74,9 @@ protected override void Setup(FeatureConfigurationContext context)
 #pragma warning restore CS0618
 #pragma warning restore IDE0079
 
+                    testContext.LocalQueueAddress = context.Settings.LocalQueueAddress();
+                    testContext.InstanceSpecificQueueAddress = context.Settings.InstanceSpecificQueueAddress();
```

Comment:
should we even capture that earlier to ensure that the API works at any point in time? e.g. at line 60 when we configure the endpoint?

Reply:
ah wait, the API would currently throw if you try to call that via `endpointConfiguration.GetSettings()` ?